node-util/sbin/oo-accept-node

```diff
@@ -455,10 +455,11 @@ def check_cgroup_procs
 
     verbose("determining node uid range: #{min_uid} to #{max_uid}")
 
-    all_user_procs = %x[/bin/ps -e -o uid,pid].split("\n")
-    ps_procs = Hash.new{|hash, key| hash[key] = Array.new}
-    all_user_procs.each do |line|
-        uid,pid = line.split[0,2]
+    # Bug 1067107 - Check tids against tasks instead of pids against cgroup procs
+    all_user_threads = %x[/bin/ps -e H -o uid,pid,tid].split("\n")
+    ps_threads = Hash.new{|hash, key| hash[key] = Hash.new{|hash, key| hash[key] = ""}}
```

Comment:
I don't see a reason for the inner Hash's initializer, `{|hash, key| hash[key] = ""}`.  If there is some reason you need it, maybe you could rename the parameters `hash` and `key` to, say, `h` and `k` to avoid shadowing the outer initializer's parameters.